Core/ContentBlockerUserScript.swift

```diff
@@ -58,7 +61,8 @@ public class ContentBlockerUserScript: NSObject, UserScript {
         return Self.loadJS("contentblocker", from: Bundle.core, withReplacements: [
             "${unprotectedDomains}": unprotectedDomains,
             "${trackerData}": trackerData,
-            "${surrogates}": surrogates
+            "${surrogates}": surrogates,
+            "${blockingEnabled}": PrivacyConfigurationManager.shared.privacyConfig.isEnabled(featureKey: .contentBlocking).description
```

Comment:
Accessing `description` or calling `String(describing:)` is actually discouraged in Swift - better explicitly use `isTrue ? "true" : "false"`